scaffold-templates/README.md

```diff
@@ -2,41 +2,42 @@
 
 We know that we can learn from a wide variety of communities, including those who will use or will be impacted by the tool, who are experts in technology, or who have experience with similar technologies deployed in other spaces. We are dedicated to creating forums for continuous conversation and feedback to help shape the design and development of the tool.
 
 We also recognize capacity building as a key part of involving a diverse open source community. We are doing our best to use accessible language, provide technical and process documents in multiple languages, and offer support to community members with a wide variety of backgrounds and skillsets. 
 
 ### Community Guidelines
-Principles and guidelines for participating in our open source community are can be found in COMMUNITY_GUIDELINES.md. Please read them before joining or starting a conversation in this repo or one of the channels listed below. All community members and participants are expected to adhere to the community guidelines and code of conduct when participating in community spaces including: code repositories, communication channels and venues, and events. 
+Principles and guidelines for participating in our open source community are can be found in [COMMUNITY_GUIDELINES.md](scaffold-templates/COMMUNITY_GUIDELINES.md). Please read them before joining or starting a conversation in this repo or one of the channels listed below. All community members and participants are expected to adhere to the community guidelines and code of conduct when participating in community spaces including: code repositories, communication channels and venues, and events. 
 
 
 ## Governance
-Information about how the {project_name} community is governed may be found in GOVERNANCE.md
+Information about how the **{project_name}** community is governed may be found in [GOVERNANCE.md](scaffold-templates/GOVERNANCE.md).
 
 ## Glossary
-Information about terminology and acronyms used in this documentation may be found in GLOSSARY.md. 
+Information about terminology and acronyms used in this documentation may be found in [GLOSSARY.md](scaffold-templates.GLOSSARY.md). 
 
 ## Feedback
-If you have ideas for how we can improve or add to our capacity building efforts and methods for welcoming people into our community, please let us know at {contact email}. If you would like to comment on the tool itself, please let us know by filing an issue on our GitHub repository.
+If you have ideas for how we can improve or add to our capacity building efforts and methods for welcoming people into our community, please let us know at **{contact email}**. If you would like to comment on the tool itself, please let us know by [filing an issue on our GitHub repository](https://github.com/DSACMS/repo-scaffolder/issues/new/choose).
```

Comment:
Same idea/pattern here of not linking directly to reposcaffolder, but to the local repo by doing something like: `[filing an issue on our GitHub repository](**REPO_URL**/issues/new/choose)`